src/main/java/frc/robot/input/SingleControllerInput.java

```diff
@@ -38,7 +48,31 @@ public double getTurnRequest() {
 
     @Override
     public boolean getShouldUseSlowSpeed() {
-        return controller.getLeftBumper();
+        return controller.getLeftStickButton();
+    }
+
+    @Override
+    public boolean getShouldMaintainWristParallel() {
+        return controller.getAButton();
     }
 
+    @Override
+    public JoystickButton getRaiseArmButton() {
+        return this.raiseArm;
+    }
+
+    @Override
+    public JoystickButton getLowerArmButton() {
+        return this.lowerArm;
+    }
+
+    @Override
+    public JoystickButton getRetractWristButton() {
+        return this.retractWrist;
+    }
+
+    @Override
+    public JoystickButton getExtendWristButton() {
+        return this.extendWrist;
+    }
```

Comment:
What's the rationale behind making these boolean inputs instead of analog inputs? We still have a couple of analog inputs left on the controller: the right y-axis and the shoulders. Wouldn't the increased precision of analog input help the driver more accurately position the shoulder/wrist?